packages/abi-typegen/src/templates/contract/main.ts

```diff
@@ -30,21 +31,26 @@ export function renderDtsTemplate(params: { abi: Abi }) {
   const { imports } = formatImports({
     types,
     baseMembers: [
-      'Interface',
+      // 'Interface',
```

Comment:
You can change the templates to just `import { ... } from 'fuels'` and then make this list like this:
`[Interface, Contract, type DecodedValue, type FunctionFragment,  type BytesLike, type InvokeFunction]`.

Regardless, this commented-out code should either be removed or uncommented and appropriately handled.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2824/commits/140889805e8c6366f2c740d04f81d1c4ce3ee96a